utils.ts

```diff
@@ -37,7 +47,12 @@ export function generateHotKeyText(hotkey: Hotkey): string {
         hotKeyStrings.push(MODIFIER_ICONS[mod])
     }
 
-    hotKeyStrings.push(hotkey.key.toUpperCase())
+    var key = hotkey.key.toUpperCase();
+    SPECIAL_KEYS.forEach (special_key =>{
+        if (special_key[0] === key) key = special_key[1];
+    });
+
+    hotKeyStrings.push(key)
```

Comment:
With the dict above we can also clean this up a little. Also it tends to be better to use `let` over `var`. `var` gets hoisted which has caused my grief in the past. In our case here though we can just use const.

```suggestion
    const key = hotkey.key.toUpperCase();
    hotKeyStrings.push(SPECIAL_KEYS[key] || key)
```